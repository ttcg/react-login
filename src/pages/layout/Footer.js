import React from 'react';
import {
    Container
} from 'react-bootstrap';
import ReactMoment from 'react-moment'

export default function Footer() {
    return (
        <footer className="fixed-bottom footer">
            <Container>
                <span className="text-muted">&copy; <ReactMoment format='YYYY' /> TTCG</span>
            </Container>
        </footer>
    );
}