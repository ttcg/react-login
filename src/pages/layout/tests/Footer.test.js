import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Footer from '../Footer';

it('should render Footer', () => {
    const component = shallow(<Footer />);
    expect(toJson(component)).toMatchSnapshot()
});