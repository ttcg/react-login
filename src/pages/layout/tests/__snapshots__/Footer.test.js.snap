// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Footer 1`] = `
<footer
  className="fixed-bottom footer"
>
  <Container
    fluid={false}
  >
    <span
      className="text-muted"
    >
      Â© 
      <t
        ago={false}
        calendar={false}
        decimal={false}
        element={null}
        filter={[Function]}
        format="YYYY"
        fromNow={false}
        interval={60000}
        local={false}
        onChange={[Function]}
        titleFormat=""
        toNow={false}
        unit={null}
        unix={false}
        utc={false}
        withTitle={false}
      />
       TTCG
    </span>
  </Container>
</footer>
`;
